Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'rails', '~> 6.1.5'
 
 gem 'bootsnap', '>= 1.4.4', require: false # https://rubygems.org/gems/bootsnap
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rails
+gem 'discard', '~> 1.2'
```

Comment:
check alphabetic order on gem :p

Reply:
@fabiianZ please don't forget to change the format (with the link added in a commentary)